django_app/redbox_app/redbox_core/models.py

```diff
@@ -59,6 +59,10 @@ class File(UUIDPrimaryKeyBase, TimeStampedModel):
     original_file = models.FileField(storage=settings.STORAGES["default"]["BACKEND"])
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     original_file_name = models.TextField(max_length=2048, blank=True, null=True)
+    core_file_uuid = models.UUIDField(null=False)
```

Comment:
```suggestion
    core_file_uuid = models.UUIDField(null=True, blank=True)
```
otherwise this migration will fail on exiting `File` records

Reply:
There are two migrations. The first adds a default value, the second make the field non-nullable, so it should work?

Reply:
afraid not, you will need an intermediate migration to populate the missing values